src/radical/pilot/agent/bootstrap_0.sh

```diff
@@ -1527,7 +1527,7 @@ PB1_LDLB="$LD_LIBRARY_PATH"
 #        We should split the parsing and the execution of those.
 #        "bootstrap start" is here so that $PILOT_ID is known.
 # Create header for profile log
-if ! test -z "$RADICAL_PILOT_PROFILE"
+if ! test -z "$RADICAL_PILOT_PROFILE$RADICAL_PROFILE"
```

Comment:
Why are you checking twice for the environment variable?

Reply:
This checks if either `RADICAL_PILOT_PROFILE` or `RADICAL_PROFILE` is set.  Both are valid by now (per RU changes), so either one should trigger profiling.